README.md

```diff
@@ -84,52 +105,82 @@ For 16B, 32B, 64B, and 128B models. See full run configs in [MaxText/configs/v5e
 
 # Features and Diagnostics
+
+Install the [Cloud TPU diagnostics](https://pypi.org/project/cloud-tpu-diagnostics) Python package to monitor, debug and profile jobs running on Cloud TPUs.  
+
 ## Collect Stack Traces
-When running a Single Program, Multiple Data (SPMD) job on accelerators, the overall process can hang if there is any error or any VM hangs/crashes for some reason. In this scenario, capturing stack traces will help to identify and troubleshoot the issues for the jobs running on TPU VMs.
 
-The following configurations will help to debug a fault or when a program is stuck or hung somewhere by collecting stack traces. Change the parameter values accordingly in `MaxText/configs/base.yml`:
-1. Set `collect_stack_trace: True` to enable collection of stack traces on faults or when the program is hung. This setting will periodically dump the traces for the program to help in debugging. To disable this, set `collect_stack_trace: False`.
-2. Set `stack_trace_to_cloud: False` to display stack traces on console. `stack_trace_to_cloud: True` will create a temporary file in `/tmp/debugging` in the TPUs to store the stack traces. There is an agent running on TPU VMs that will periodically upload the traces from the temporary directory to cloud logging in the gcp project. You can view the traces in Logs Explorer on Cloud Logging using the following query:
-```
-logName="projects/<project_name>/logs/tpu.googleapis.com%2Fruntime_monitor"
-jsonPayload.verb="stacktraceanalyzer"
-```
-3. `stack_trace_interval_seconds` signifies the duration in seconds between each stack trace collection event. Setting `stack_trace_interval_seconds: 600` will collect the stack traces every 600 seconds (10 minutes).
+When running a Single Program, Multiple Data (SPMD) job on accelerators, the overall process can hang if any errors occur, or if a VM hangs or crashes. Capturing stack traces will help you identify and troubleshoot the issues that are occuring.
+
+The following configurations will help you debug a workload by collecting stack traces. Change the following parameter values accordingly in `MaxText/configs/base.yml`:
+
+* Set `collect_stack_trace: True` to enable collection of stack traces on faults 
+  or when the program hangs. This setting periodically dumps stack traces. To 
+  disable this, set `collect_stack_trace: False`.
+
+* Set `stack_trace_to_cloud: False` to display stack traces on the console. 
+  Or, set `stack_trace_to_cloud: True` to create a temporary file in `/tmp/debugging` 
+  in your TPU VMs to store stack traces. There is an agent running on TPU VMs that 
+  periodically uploads traces from the temporary directory to [Cloud Logging](https://cloud.google.com/logging/docs/overview). 
+  You can view the traces in [Logs Explorer](https://cloud.google.com/logging/docs/view/logs-explorer-interface) 
+  in Cloud Logging using the following query:
+
+   ```none
```

Comment:
Any keyword after ``` is used to define what language is used in the following code block. Since I didn't know if the code block was specific to any language, I used "none". If yaml would be more appropriate I could use that.